src/VuData.cpp

```diff
@@ -1245,9 +1269,24 @@ PVR_ERROR Vu::GetInitialEPGForChannel(ADDON_HANDLE handle, const VuChannel &chan
       PVR->TransferEpgEntry(handle, &broadcast);
     }
   }
+
   return PVR_ERROR_NO_ERROR;
 }
 
+bool Vu::CheckIfAllChannelsHaveInitialEPG() const
```

Comment:
This function could be static as it does. not operate on any members of Vu class.